topics/go/algorithms/strings/reverse/reverse.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+
+	fmt.Println(ReverseString("Hello World"))
+}
+
+func ReverseString(str string) string {
+
+	// Here we create a slice of codePoints.
+	codePoints := []rune(str)
+
+	// Here we create int that will be a pointer to the front of the codePoints.
+	var fontCodePoint int
```

Comment:
Now that the slice is named right, these variable names are redundant. How do we shorten them?

I like `end` so what’s good for the other?

`top`
`beg`

Reply:
I would say beg

Reply:
Ok